packages/relay/src/lib/services/ethService/ethCommonService/index.ts

```diff
@@ -334,7 +335,7 @@ export class CommonService implements ICommonService {
         new Log({
           address: log.address,
           blockHash: toHash32(log.block_hash),
-          blockNumber: numberTo0x(log.block_number),
+          blockNumber: nullableNumberTo0x(log.block_number),
```

Comment:
Yes, this is one of the strange cases where `block_number`, `transaction_index`, or `blockHash` are falsy, leading to 500 errors. The retry mechanism should address the issue. Should I revert this change and throw a more informative error message even if those fields remain falsy after the retry? Hmm maybe that would be better so the clients can resubmit the requests themselves other than having null in the response. Wdyt?

Reply:
Hmmm, if a retry is ensured to resolve the issue then maybe we can manage one iteration in the short term.
We should open up an issue in the mirror node repo to investigate this issue so this can be resolved.